package.json

```diff
@@ -9,8 +9,9 @@
     "build:favicons": "node scripts/favicons",
     "build": "yarn run build:dir && yarn run build:favicons && gatsby build --prefix-paths --log-pages && yarn updateRelativePath",
     "build:incremental": "gatsby build --prefix-paths --log-pages && yarn updateRelativePath",
+    "cleandev": "gatsby clean && gatsby develop",
     "devbuild": "gatsby clean && yarn run build:dir && yarn run build:favicons && gatsby build",
-    "develop": "gatsby develop",
+    "dev": "gatsby develop",
```

Comment:
This is a good change, but check the README doesnt mention the old command

Reply:
Sorry I missed that comment before merhing.
You're right, it does mention "yarn develop" 
I will update the ReadMe in a new PR 👍